app/models/user/UserCurrentRegionTable.scala

```diff
@@ -87,47 +115,28 @@ object UserCurrentRegionTable {
   def assignNextRegion(userId: UUID): Int = db.withSession { implicit session =>
     val regionIds: Set[Int] = MissionTable.selectIncompleteRegions(userId)
 
+    // TODO: Add a detailed comment
     val difficultRegionCompletions: List[RegionCompletion] =
       RegionCompletionTable.regionCompletions
         .filter(_.regionId inSet regionIds)
         .filter(_.regionId inSet difficultRegionIds)
         .filter(region => region.auditedDistance / region.totalDistance < 1.0)
         .sortBy(region => region.auditedDistance / region.totalDistance).list
 
-    // if they have audited less than 2 miles and there is an easy region left (or if there are no difficult regions
+    // If they have audited less than 2 miles and there is an easy region left (or if there are no difficult regions
     // left to finish), give them an easy one
     if ((regionIds.filterNot(difficultRegionIds.contains(_)).nonEmpty && !isUserExperienced(userId)) ||
         difficultRegionCompletions.isEmpty) {
-      assignNextEasyRegion(userId)
+      assignEasyRegion(userId)
     }
     else {
-      // take the least-audited difficult region
+      // Take the least-audited difficult region
```

Comment:
Also, this code doesn't seem to be called since registered users who are experienced are getting an easy region after finishing their previous region.